### DISCUSSION TITLE
Multi iif question

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#1091] [General]
Greetings

Is there a way in infinity to do multiple conditionals?

**Comment by yesoreyeram** [comment]
Can you give me example what you are after..?

**Comment by yosiasz** [comment]
Something like a switch statement in js

```
switch (x) {
  case 'Monday':
    Sort = 1 ;
```

For example you have weekdays you need to sort by. You can't use the weekdays string so you use a switch to assign these weekday names to a numeric value then sort on that extended value

Something like jsonata that extends things but for CSV

Thanks
